Tools/Scripts/libraries/resultsdbpy/resultsdbpy/controller/api_routes.py

```diff
@@ -77,6 +81,10 @@ def __init__(self, model, import_name=__name__, auth_decorator=None):
         self.add_url_rule('/urls/queue', 'queue-urls', self.ci_controller.urls_for_queue_endpoint, methods=('GET',))
         self.add_url_rule('/urls', 'build-urls', self.ci_controller.urls_for_builds_endpoint, methods=('GET',))
 
+        if bug_tracker_configs:
+            self.add_url_rule('/bug-trackers', 'bug-trackers', self.bug_tracker_controller.list_trackers, methods=('GET',))
+            self.add_url_rule('/bug-trackers/<path:tracker>/create-bug', 'create-bug', self.bug_tracker_controller.create_bug, methods=('POST',))
```

Comment:
This is because for Radar we can return an app url, but for bugzilla, we can only use the backend API to create the bug first and then return the url.